src/omv/imlib/imlib.c

```diff
@@ -17,17 +17,21 @@
 #include "file_utils.h"
 #include "imlib.h"
 #include "omv_common.h"
+#include "omv_gpu.h"
 #include "omv_boardconfig.h"
 
 void imlib_init_all() {
+    #if (OMV_GPU_ENABLE == 1)
+    omv_gpu_init();
+    #endif
```

Comment:
```suggestion
```
This should be called from main(), as mentioned before. I may need to call it once on first soft-reset.